metagraph-plugin/conda_build_dir/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "metagraph-katana" %}
+{% set version = "0.0.5" %}
```

Comment:
I would prefer picking up the version from the main katana build. That's tricky though (see the `KATANA_VERSION` hack in the other packages), so this isn't high priority.